Quotient/room.h

```diff
@@ -721,8 +721,28 @@ class QUOTIENT_API Room : public QObject {
         return post(makeEvent<EvT>(std::forward<ArgTs>(args)...));
     }
 
+    /// Send a plain text message
+    QString postPlainText(const QString& plainText, std::optional<EventRelation> relatesTo = std::nullopt);
+
+    /// Send a rich text message
+    QString postHtmlText(const QString& plainText, const QString& html, std::optional<EventRelation> relatesTo = std::nullopt);
+
+    /// Send a m.emote message
+    QString postEmote(const QString& plainText, std::optional<const QString> html = std::nullopt, std::optional<EventRelation> relatesTo = std::nullopt);
+
+    /// Send an m.notice message
+    QString postNotice(const QString& plainText, std::optional<const QString> html = std::nullopt, std::optional<EventRelation> relatesTo = std::nullopt);
```

Comment:
Might I suggest making it a single templated inline member function, with the non-type template parameter defaulting to `MessageEventType::Text`? You can static-assert the correct msgtype in it. And now that I think of it, perhaps `postPlainText()` and `postHtmlText()` can be merged into a single `postText()` function, because the original distinction between the two was made solely for the purpose of accessibility from QML, back in Spectral times.